### DISCUSSION TITLE
Yimmenu: [Upload XML: files]

### DISCUSSION THREAD

**Discussion by mvcula** [YimMenu/YimMenu#2715] [General]
Hello, anyone can tell me how upload xml vehicles and maps? where i find something.

**Comment by 99Anvar99** [comment]
> Hello, anyone can tell me how upload xml vehicles and maps? where i find something.

Appdata->YimMenu There u should see folders named XML Vehicles and maps

Edit: do you mean where you put the files to load it in the menu?

If you are looking for some XML files you can mostly find them on GTA5 Mods website.

**Reply by mvcula** [reply]
i had some xml vehicles etc n reload lua in menu but when i trying spawn car, game crashing

**Reply by 99Anvar99** [reply]
It's because you are not using the latest build, I have already fixed this issue but the nightly bot didn't update the DLL.

Edit: Try the commit from https://github.com/YimMenu/YimMenu/pull/2707 (The DLL should be emplaced from the builder bot in the ticket)

**Reply by Yimura** [reply]
If the PR is merged then the YimMenu builder should've update the DLL appropriately. Do note that the bot only runs once a day around 3PM UTC.

**Reply by mvcula** [reply]
Cool, thats working. Thanks alot m8 Greetings!

**Reply by Yimura** [reply]
Yeah it should be able to just fine, however ideally we’d setup some cache so we don’t need to gather the dependencies everytime.
